articles/api-auth/config/using-the-auth0-dashboard.md

```diff
@@ -0,0 +1,32 @@
+---
+description: How to enable API Authorization on the Auth0 dashboard. 
+---
+
+# API Authorization: Using the Auth0 Dashboard
+<%=include('../_preview-warning') %>
+
+1. Open the [Auth0 Dashboard](${uiURL}) and browse to the Clients section.
+
+2. Create a new client of type **Non Interactive** for each of the applications that will consume the API you want to generate access tokens for.
+
+3. Navigate to the API section and create a new API by entering a friendly name and identifier (this would be the public endpoint of the API). This API will represent your resource server.
```

Comment:
Ideally the identifier should be the public endpoint of the API but this is not mandatory. It is ok to submit any valid urn as identifier